### DISCUSSION TITLE
Is rate limiter local ? or global ?

### DISCUSSION THREAD

**Discussion by chenxxzhe** [OptimalBits/bull#2474] [Q&A] [answered]
There are many servers running jobs, use the same queue, such as `new Bull('task', {})`
If I set 
```
limiter: {max: 5, duration: 10000}
```
Does all servers only complete max 5 tasks during 10s  (global)? Or each server complete max 5 tasks during 10s (local).

If limiter is global, how can I set local limiter ?

thanks.

**Comment by manast** [comment, marked as answer]
The rate limiter is global for a given Queue, I wrote a blog post about rate-limiting that may be helpful to you (it is for BullMQ but the basics are the same): https://blog.taskforce.sh/rate-limit-recipes-in-nodejs-using-bullmq/
